templates/bplink.html

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
+<html lang="ja">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link rel="icon" href="">
+
+    <title>The NBDC-DDBJ imputation server bplink2vcf conversion configuration</title>
+
+    <!-- Bootstrap core CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+    <!-- Custom styles for this template -->
+    <link href="https://getbootstrap.com/docs/4.0/examples/offcanvas/offcanvas.css" rel="stylesheet">
+
+    <style>
+        body {
+  font-family: 'Inter', 'Segoe UI', 'Roboto', 'Droid Sans', sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+        label {
+            margin-top: 0.2rem;
+        }
+    </style>
+</head>
+
+<body class="bg-light">
+
+    <form enctype="multipart/form-data" action="/" method="POST" id="wholegenomeform">
```

Comment:
```suggestion
    <form enctype="multipart/form-data" action="/bplink" method="POST" id="wholegenomeform">
```
ブラウザでボタンが押された後の飛び先を、 `/bplink` にしていただけますか？